common/test/acceptance/tests/studio/test_studio_course_team.py

```diff
@@ -26,6 +26,21 @@ def _make_user(self, username):
         ).visit()
         return user
 
+    def _update_user(self, **update_values):
+        """
+        Update other_user with provided values
+
+        Arguments:
+            `update_values`: dictionary containing values of attributes to be updated
+        """
+        AutoAuthPage(
+            self.browser, no_login=True, username=self.other_user.get('username'),
+            email=self.other_user.get('email'), password=self.other_user.get('password'),
```

Comment:
This method will only be updating `self.other_user`, lets make it such that we can update user provided in `user_info`.